### ISSUE TITLE
Datagrid column sorting changing information is not read by screen reader

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#6498]
## Describe the bug

When the column header is clicked, the sorting changing information is not read by screen reader

## How to reproduce

1.  Go to '[stackblitz clarity datagrid simple sample](https://stackblitz.com/edit/clarity-light-theme-v5-datagrid-simple)'
2.  Run a screen reader such as VoiceOver for MacOS or NVDA for Windows
3.  Click on datagrid 'name' column header
4.  The sorting of name column is changed, but nothing read by the screen reader

## Expected behavior

The sorting changing information should be read by the screen reader.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**

Angular 11

**Comment by bdryanovski** [comment]
We had such type of functionality a few versions ago, but remove it because it was the introduction a lot of complexity to the UI and did not always was working as people expected. 

As a workaround, I could suggest using the `aria-live` attribute to announce the change. Our policy for v5 was that Aria Live events will be a developer's responsibility - for the reason mentioned above.
